src/image.ts

```diff
@@ -49,6 +49,16 @@ export interface ImageProps {
    */
   readonly file?: string;
 
+  /**
+   * Name for tagging the image
+   */
+  readonly name?: string;
+
+  /**
+   * Tag for tagging the image
+   */
+  readonly tag?: string;
```

Comment:
The description of these fields might be confusing to someone less familiar with how docker tagging works. From what I see at https://docs.docker.com/engine/reference/commandline/tag/ it looks like the convention is `{registry}/{name}:{tag}` -- maybe we could mention that in the descriptions, or link to those docs explicitly?

Let's also update the docstrings to mention what the defaults are:

```
@default - auto-generated name
...
@default "latest"
```

Reply:
Got you
just updated!